components/datepicker/modules/month-year-picker.vue

```diff
@@ -140,7 +156,64 @@ export default {
 
   },
 
+  mounted () {
+    this.focusMonthYearPicker();
+  },
+
   methods: {
+    formatMonth (month, monthFormat) {
+      return format(new Date(2000, month, 1), monthFormat);
+    },
+
+    setDayRef (el) {
+      if (!this.focusRefs.includes(el)) {
+        this.focusRefs.push(el);
+      }
+    },
+
+    focusMonthYearPicker () {
+      this.focusRefs[0].focus();
+    },
+
+    handleKeyDown (event) {
+      switch (event.key) {
+        case 'ArrowLeft':
+          event.preventDefault();
+          if (this.focusPicker === 0) {
+            this.focusPicker = 3;
+            this.focusRefs[this.focusPicker].focus();
+          } else {
+            this.focusPicker--;
+            this.focusRefs[this.focusPicker].focus();
+          }
+          break;
+
+        case 'ArrowRight':
+          event.preventDefault();
+          if (this.focusPicker === 3) {
+            this.focusPicker = 0;
+            this.focusRefs[this.focusPicker].focus();
+          } else {
+            this.focusPicker++;
+            this.focusRefs[this.focusPicker].focus();
+          }
+          break;
+
+        case 'ArrowDown':
+          event.preventDefault();
+          this.$emit('focus-day');
+          break;
```

Comment:
Also this shouldn't change the focus to the calendar, It should only change with `tab`

Reply:
same answer as the above, and also it is doing it with `tab`